pages-css/compo-css/navbar.css

```diff
@@ -0,0 +1,68 @@
+       /* color variables */
+       
+        :root {
+           --theam-color-black: #000000;
+           --theam-color-orange: #ffa245;
+           --color-light-orange: #fff2d9;
+       }
+       /* *** */
+       /* simple nav-bar */
+       
+       .nav-bar {
+           display: flex;
+           height: 1.875rem;
+           width: 100%;
+           align-items: center;
+           justify-content: space-between;
+           margin-left: 1.25rem;
+           margin-right: 1.25rem;
```

Comment:
this can be clubbed as margin: 0 1.25rem;

Reply:
Ok